server/search/actions_index.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package search
+
+import (
+	"context"
+
+	registry_rpc "github.com/apigee/registry/rpc"
+
+	experimental_rpc "github.com/apigee/registry-experimental/rpc"
```

Comment:
Outside of the scope of this PR, but we should consider having separate Go packages for our protos. I think ideally our proto imports for this file would be something like the following.

```
regpb "github.com/apigee/registry/google/cloud/apigeeregistry/v1/registry_go_proto"
expb "github.com/apigee/registry-experimental/google/cloud/apigeeregistry/v1/search_go_proto"
```

It would make the imports more descriptive, reduce the amount of unrelated code being imported, colocate the protos with their generated packages, and increase the definition of the boundaries between components of the system.